proxy/proxymanager.go

```diff
@@ -387,6 +389,27 @@ func (pm *ProxyManager) unloadAllModelsHandler(c *gin.Context) {
 	c.String(http.StatusOK, "OK")
 }
 
+func (pm *ProxyManager) running(context *gin.Context) {
+	context.Header("Content-Type", "application/json")
+	response := gin.H{} // Default to an empty JSON object
```

Comment:
llama-swap can run multiple processes when using the profiles feature so the response type should be: 

```
{
   "running": [
       "modelname": state
   ]
}
```